src/components/list-item/list-item.tsx

```diff
@@ -431,6 +501,7 @@ export class ListItem implements InteractiveComponent, LoadableComponent {
             [CSS.containerBorderSelected]: borderSelected,
             [CSS.containerBorderUnselected]: borderUnselected
           }}
+          hidden={closed}
```

Comment:
We should consolidate how we handle closed/removed items. I've noticed we have the following discrepancies:

1. naming: `closable` vs `removable`
2. approach: hiding vs removing the node

I'll create a separate issue for this.

Reply:
I don't think we use removable anywhere else besides value-list, right?

I think sometimes we can't remove the node because we need the slot to still be there so we hide it.